src/templates/ObservabilityPackDetails.js

```diff
@@ -124,124 +274,25 @@ const ObservabilityPackDetails = ({ data, location }) => {
-                </Intro>
-                {pack.dashboards?.map((dashboard) => (
-                  <>
-                    <h3>{dashboard.name}</h3>
-                    <ImageSlider height={400} images={dashboard.screenshots} />
-                    {dashboard.description && (
-                      <>
-                        <h4>Description</h4>
-                        <p>{dashboard.description}</p>
-                      </>
-                    )}
-                  </>
-                ))}
+                {pack.dashboards
+                  ? renderDashboards(pack.dashboards)
+                  : emptyStateContent(pack, 'dasboards')}
               </Tabs.Page>
               <Tabs.Page id="alerts">
-                <Intro
-                  css={css`
-                    margin-bottom: 16px;
-                  `}
-                >
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
-                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-                  irure dolor in reprehenderit in voluptate velit esse cillum
-                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-                  cupidatat non proident, sunt in culpa qui officia deserunt
-                  mollit anim id est laborum.
-                </Intro>
-                {pack.alerts?.map((alert) => (
-                  <>
-                    <h3>{alert.name}</h3>
-                    {alert.description && (
-                      <>
-                        <h4>Description</h4>
-                        <p>{alert.description}</p>
-                      </>
-                    )}
-                  </>
-                ))}
+                {pack.alerts
+                  ? renderAlerts(pack.alerts)
+                  : emptyStateContent(pack, 'alerts')}
               </Tabs.Page>
               <Tabs.Page id="synthetics">
-                <Intro
-                  css={css`
-                    margin-bottom: 16px;
-                  `}
-                >
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
-                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute
-                  irure dolor in reprehenderit in voluptate velit esse cillum
-                  dolore eu fugiat nulla pariatur. Excepteur sint occaecat
-                  cupidatat non proident, sunt in culpa qui officia deserunt
-                  mollit anim id est laborum.
-                </Intro>
+                {pack.synthetics
+                  ? renderSynthetics()
+                  : emptyStateContent(pack, 'Synthetics checks')}
```

Comment:
Just wondering if this should be `Synthetics checks` or just `Synthetics`, I know we recently changed that for the tab itself

Reply:
I thought this was a bit odd, but my understanding is this is the correct wording for the product.